nautobot/extras/forms/forms.py

```diff
@@ -967,6 +973,13 @@ def __init__(self, *args, **kwargs):
 
 
 class JobEditForm(NautobotModelForm):
+    job_queues = DynamicModelMultipleChoiceField(
+        label="Job Queues",
+        queryset=JobQueue.objects.all(),
+        required=False,
+        widget=APISelectMultiple(api_url="/api/extras/job-queues/"),
```

Comment:
Doesn't/shouldn't the widget work automatically for a `DynamicModelMultipleChoiceField`? Why do we need to explicitly specify it here?

Reply:
Removed